### DISCUSSION TITLE
- An Analysis of Seasonality on Mass Shootings

### DISCUSSION THREAD

**Discussion by giscus** [DACSS/DACSS_601_August2022_v2#255] [Announcements]
# - An Analysis of Seasonality on Mass Shootings

Final Project

https://dacss.github.io/DACSS_601_August2022_v2/posts/FinalProject_EmmaRasmussen.html

**Comment by LarriMill** [comment]
Hi Emma, this is really thorough work. I am very impressed.

**Reply by mrolfe** [reply]
Agreed - excellent project. I really like the way that you compared Mass/Ill to Florida to tease out some of the temperature/seasonality issues. I can imagine that you could work in additional variables (temperature, gun ownership, etc) in a future study that could be a lot of fun.  The severity analysis was really eye-opening to me, because it suggests that the major pre-meditated shootings are likely not responsive to the seasonality trends, while perhaps more spur of the moment, opportunistic shootings (rage + opportunity) are the ones responsive to seasonality.

One area where I think you could have made things a little easier would have been to rely more on `lubridate` to handle date management. For example `month(incident_date, label=TRUE,abbr=TRUE)` would have gotten you month abbreviations for graphing without quite so much manual coding, but otherwise looks good.
